lib/setExtensions.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * A set of useful extensions to the Set prototype.
+ */
+
+Set.prototype.union = function(other) {
```

Comment:
Mostly so I could use normal set semantics, but I could definitely rewrite it to just use arrays and lodash.